__tests__/Equality.ts

```diff
@@ -12,14 +12,18 @@ describe('Equality', () => {
 
   function expectIs(left, right) {
     var comparison = Immutable.is(left, right);
+    expect(comparison).toBe(true);
     var commutative = Immutable.is(right, left);
-    return comparison && commutative && comparison === commutative;
+    expect(commutative).toBe(true);
+    expect(comparison === commutative).toBe(true);
```

Comment:
this line isnt necessary since you already made the expectations above that both values are `true`, which means there is no possible scenario that this expectation could fail.